src/components/__tests__/ComingSoon.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env jest */
+
+import React from 'react'
+import { mount, shallow } from 'enzyme'
+import Typography from '@material-ui/core/Typography'
+import data from 'src/data/causes/cats.json'
+import Countdown from 'react-countdown'
+const getMockProps = () => ({
+  location: {
+    pathname: '/',
+  },
+  pageContext: {
+    data: data.data,
+  },
+})
+
+describe('ComingSoon page component', () => {
+  it('renders without error', () => {
+    const ComingSoon = require('../ComingSoon').default
+    shallow(<ComingSoon {...getMockProps()} />)
+  })
+
+  it('sets the title correctly according to the cause data', async () => {
+    const mockProps = getMockProps()
+    const ComingSoon = require('../ComingSoon').default
+    const wrapper = mount(<ComingSoon {...mockProps} />)
+    const typography = wrapper.find(Typography).first()
+    expect(typography.text()).toEqual(
+      `TAB FOR ${mockProps.pageContext.data.sections.comingSoon.name}`
+    )
+  })
+
+  it('sets the date in Countdown Component correctly according to the cause data', async () => {
+    const mockProps = getMockProps()
+    const ComingSoon = require('../ComingSoon').default
+    const wrapper = mount(<ComingSoon {...mockProps} />)
+    const countdown = wrapper.find(Countdown).first()
+    expect(countdown.props().date).toEqual(
+      mockProps.pageContext.data.sections.comingSoon.date
+    )
+  })
```

Comment:
Add tests for these ACs:
* If no launch time is specified, just say “Coming soon” with no countdown
* If launch time has passed but we haven’t launched, just say “Coming soon” with no countdown